### ISSUE TITLE
Generate an index for a particular tag

### ISSUE THREAD

**Issue by evantravers** [vimwiki/vimwiki#724]
This is just an idea… I sometimes manage "hub" pages in my wiki, and it'd be nice to be able to have a live updating list of pages with just specific tags rather than all tags… something like

```
# Generated Tag :foobar:
```

And that would pull only that tag… sorry if this is a duplicate, I didn't see one.

**Comment by evantravers** [comment]
I'm an idiot. `:VimwikiGenerateTags` does take arguments. For those who find this via google…  it seems that on the `dev` branch, the `auto_tags` and `auto_generate_tags` options will cause them to auto update. Pretty close to ideal for my uses.

**Comment by evantravers** [comment]
Well… `auto_generate_tags` causes the magic header for a single set of tags that you wish to track to automatically expand to _all_ tags.

We may want a magic header like the one I described in the first comment.

**Comment by ranebrown** [comment]
I agree that could be useful. Not sure when anyone will have time to look into it though. If you want to create a PR that would definitely speed things up.
